[HttpStress] [SslStress] Windows stress builds are broken

```
2023-12-07T13:23:15.1583259Z   D:\a\_work\1\s\src\native\corehost\test\ijw\ijw.cpp(22): error C3611: 'System::Numerics::INumberBase::System::IUtf8SpanFormattable.TryFormat': a sealed function cannot have a pure-specifier
```

The failures started on the 17th of November, across all branches, which indicates an image change might have triggered this, we should do a comparison similar to https://github.com/dotnet/runtime/issues/82528#issuecomment-1446676611.

*Edit: links to the pipelines*
- [http](https://github.com/dotnet/runtime/blob/8b04e76468af5905f739506e97039170b7554671/eng/pipelines/libraries/stress/http.yml#L111-L113)
- [ssl](https://github.com/dotnet/runtime/blob/8b04e76468af5905f739506e97039170b7554671/eng/pipelines/libraries/stress/ssl.yml#L63-L65)

*The failing build that runs **outside** of docker:* https://github.com/dotnet/runtime/blob/8b04e76468af5905f739506e97039170b7554671/eng/docker/build-docker-sdk.ps1#L24

--- a/eng/pipelines/libraries/stress/http.yml
+++ b/eng/pipelines/libraries/stress/http.yml
@@ -46,6 +46,9 @@ extends:
           fetchDepth: 5
 
         - bash: |
+            # Workaround for https://github.com/microsoft/azure-pipelines-agent/issues/4554. Undo when the image bug is fixed.
+            Remove-Item -Path "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\Microsoft.VCToolsVersion.v143.default.txt"
+
             $(dockerfilesFolder)/build-docker-sdk.sh -t $(sdkBaseImage) -c $(BUILD_CONFIGURATION) && \
             echo "##vso[task.setvariable variable=succeeded;isOutput=true]true"
           name: buildRuntime

--- a/eng/pipelines/libraries/stress/http.yml
+++ b/eng/pipelines/libraries/stress/http.yml
@@ -46,9 +46,6 @@ extends:
           fetchDepth: 5
 
         - bash: |
-            # Workaround for https://github.com/microsoft/azure-pipelines-agent/issues/4554. Undo when the image bug is fixed.
-            Remove-Item -Path "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\Microsoft.VCToolsVersion.v143.default.txt"
-
             $(dockerfilesFolder)/build-docker-sdk.sh -t $(sdkBaseImage) -c $(BUILD_CONFIGURATION) && \
             echo "##vso[task.setvariable variable=succeeded;isOutput=true]true"
           name: buildRuntime
@@ -122,6 +119,9 @@ extends:
           lfs: false
 
         - powershell: |
+            # Workaround for https://github.com/microsoft/azure-pipelines-agent/issues/4554. Undo when the image bug is fixed.
+            Remove-Item -Path "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\Microsoft.VCToolsVersion.v143.default.txt"
+
             $(dockerfilesFolder)/build-docker-sdk.ps1 -w -t $(sdkBaseImage) -c $(BUILD_CONFIGURATION)
             echo "##vso[task.setvariable variable=succeeded;isOutput=true]true"
           name: buildRuntime

--- a/eng/pipelines/libraries/stress/ssl.yml
+++ b/eng/pipelines/libraries/stress/ssl.yml
@@ -76,6 +76,9 @@ extends:
           lfs: false
 
         - powershell: |
+            # Workaround for https://github.com/microsoft/azure-pipelines-agent/issues/4554. Undo when the image bug is fixed.
+            Remove-Item -Path "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\Microsoft.VCToolsVersion.v143.default.txt"
+
             $(dockerfilesFolder)/build-docker-sdk.ps1 -w -t $(sdkBaseImage) -c $(BUILD_CONFIGURATION)
           displayName: Build CLR and Libraries
 

